### PR TITLE
Request to merge Jest testing

### PR DESCRIPTION
**Pull request by Georjane**
Setup tests for the project.
- Add unit tests for your calculate.js file.
- Make sure that all elements are displayed at Home page, Calculator page, Quote page

### CODE REVIEW

**Review by NtwaliHeritier — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUIRED🛑️🛑️🛑️**

Hello @Georjane 
Great work on this milestone.
This is impressive work👏️👏️

However, let's make some improvements and get it approved.

[REQUIRED]: there are snapshot tests that are checking if the main components, at least Home, Calculator, and Quote, from Math Magicians mockups are displayed.
[REQUIRED]: Add negative test cases.
[REQUIRED]: Write test cases every function in the app.

That's it.
Make the required changes and get it approved👍️👍️.
Happy coding💻️💻️

File: src/_tests_/__snapshots__/snapshots.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+<div
+  className="quote"
+>
+  <h2>
+    <blockquote>
+      Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. –William Paul Thurston
+    </blockquote>
+  </h2>
+</div>
+`;
```
Comment (Elbie-em) [inline src/_tests_/__snapshots__/snapshots.test.js.snap]: - [x] You should create separate snapshots for your integration testing other than mixing them all in one file. This makes your project look more professional with clean with readable code. e.g for the home component you can have the snapshot in `home.test.js.snap`.

File: src/_tests_/snapshots.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+it('renders quotes page', () => {
+  const tree = renderer.create(<Quote />);
+  expect(tree).toMatchSnapshot();
+});
+
+it('renders home page', () => {
+  const tree = renderer.create(<Home />);
+  expect(tree).toMatchSnapshot();
+});
```
Comment (Elbie-em) [inline src/_tests_/snapshots.test.js]: - [x] Kindly add tests for your snapshots in different test files e.g home component to be tested in `home.test.js`

File: src/_tests_/app.test.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+  it('calculates the dividend of two digits', () => {
+    const result = calculate({
+      total: '10', next: '5', operation: '÷',
+    }, '=');
+    expect(result).toEqual({
+      total: '2', next: null, operation: null,
+    });
+  });
+});
```
Comment (Elbie-em) [inline src/_tests_/app.test.js:20]: - [x] There is a mixture of integration and unit testing in this file, create `calculate.test.js ` to test all functions of your calculator and move the integration testing units to their respective files, do not forget to add negative tests for your cases.

**Review by Elbie-em — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED 🟡  ♻️ 

Hi @Georjane, great effort implementing some of the changes by the previous TSE. However, a few improvements need to be made to make your project stand out and look more professional, and have this project approved. Check my comments below ⬇️ 

*All the best, you are doing an awesome job, keep up the good work*

### *Request another code review after applying the changes*

> Elbie Moonga (Part-time TSE)

> Github (@Elbie-em)
> Slack (Elbie Moonga)
> Twitter ([@ElbieEm](https://twitter.com/ElbieEm))
> LinkedIn([Elbie Moonga](https://www.linkedin.com/in/elbiemoonga/))

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED👍️👍️👍️👍️**

Hello @Georjane 
Great work making the requested changes.👏️👏️

You can merge👍️👍️

Happy coding💻️💻️💻️

**Pull request merged**